fixtures/markdown/file1.md

```diff
@@ -78,7 +78,7 @@ reporter: json
 
 languages-exts:
     coffeescript:
-        - coffeee
+        - coffee
```

Comment:
This appears to be a fairly rare typo:

GitHub says there are 18 file hits for it: https://cs.github.com/?scopeName=All+repos&scope=&q=path%3A*.coffeee